<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abstract Spacecraft v0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/galaxy.ico</normaloff>:/img/galaxy.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Default"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionApplication_font"/>
    <addaction name="actionCollision_response"/>
    <addaction name="actionText_Collision_response"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewDiagram"/>
    <addaction name="actionNewLogicalRule"/>
    <addaction name="actionNewProof"/>
    <addaction name="actionNewWindow"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionDeleteTab"/>
    <addaction name="actionCloseWindow"/>
    <addaction name="actionCloseEntireApp"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGraphics_Debugger"/>
   </widget>
   <widget class="QMenu" name="menu_Export">
    <property name="title">
     <string>&amp;Export</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menu_Export"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="blankSpace_2"/>
   <addaction name="actionBook_Lookup"/>
   <addaction name="actionEdit_Text"/>
   <addaction name="actionPlace_Arrow"/>
   <addaction name="actionMoveStuffMode"/>
   <addaction name="blankSpace"/>
   <addaction name="actionFunctor"/>
  </widget>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_Default">
   <property name="text">
    <string>Zoom Default</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionNewProof">
   <property name="text">
    <string>New Proof</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewDiagram">
   <property name="text">
    <string>New Diagram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/navigate_back.ico</normaloff>:/img/navigate_back.ico</iconset>
   </property>
   <property name="text">
    <string>Navigate back</string>
   </property>
   <property name="toolTip">
    <string>Back   Ctrl+Backspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset>
     <normalon>:/img/navigate_forward.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Navigate forward</string>
   </property>
   <property name="toolTip">
    <string>Forward   Ctrl+Space</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionClose_Diagram">
   <property name="text">
    <string>Close diagram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionPlace_Arrow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/arrow.ico</normaloff>:/img/arrow.ico</iconset>
   </property>
   <property name="text">
    <string>Place Arrows</string>
   </property>
   <property name="toolTip">
    <string>Place arrows instead of nodes.   Alt+A</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionEdit_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/letter_T.png</normaloff>:/img/letter_T.png</iconset>
   </property>
   <property name="text">
    <string>Edit Text</string>
   </property>
   <property name="toolTip">
    <string>Double-click text to edit it.   Alt+T</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionLaTeX_to_unicode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LaTeX to unicode</string>
   </property>
  </action>
  <action name="actionBook_Lookup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/book_lookup.png</normaloff>:/img/book_lookup.png</iconset>
   </property>
   <property name="text">
    <string>Book Lookup</string>
   </property>
   <property name="toolTip">
    <string>Double-click looks up item in the library.</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionApplication_font">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Application font...</string>
   </property>
  </action>
  <action name="actionGraphics_Debugger">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/galaxy.ico</normaloff>:/img/galaxy.ico</iconset>
   </property>
   <property name="text">
    <string>Graphics Debugger</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionCollision_response">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Collision response</string>
   </property>
  </action>
  <action name="actionText_Collision_response">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text Collision response</string>
   </property>
  </action>
  <action name="actionFunctor">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/functor_of_dot.png</normaloff>:/img/functor_of_dot.png</iconset>
   </property>
   <property name="text">
    <string>F(.)</string>
   </property>
   <property name="toolTip">
    <string>Take Functorial Image Ctrl+F</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="blankSpace">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="blankSpace_2">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionMoveStuffMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/move_crosshairs.ico</normaloff>:/img/move_crosshairs.ico</iconset>
   </property>
   <property name="text">
    <string>Move Stuff</string>
   </property>
   <property name="toolTip">
    <string>Move stuff in the scene easily.</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionCloseWindow">
   <property name="text">
    <string>Close Window</string>
   </property>
  </action>
  <action name="actionDeleteTab">
   <property name="text">
    <string>Delete Tab</string>
   </property>
  </action>
  <action name="actionCloseEntireApp">
   <property name="text">
    <string>Close Entire App</string>
   </property>
  </action>
  <action name="actionNewLogicalRule">
   <property name="text">
    <string>New Logical Rule</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
